ansible_hosts

```diff
@@ -1,5 +1,5 @@
 [local]
-127.0.0.1
+localhost ansible_connection=local
 
 [gce_instances]
 myinstance[1:4]
```

Comment:
See below comment: it looks like you dropped this down to two VMs instead of 4. Check that you've removed references to `myinstance[34]`.